docs/sources/panels/field-options/standard-field-options.md

```diff
@@ -86,6 +87,20 @@ You can also paste a native emoji in the unit picker and pick it as a custom uni
 
 Grafana can sometime be too aggressive in parsing strings and displaying them as numbers. To make Grafana show the original string create a field override and add a unit property with the `string` unit.
 
+## Color
+
+> Only available in Grafana 7.3+
+
+The field color option defines how Grafana should color series or fields. There are multiple modes here that work very differently and their utility depend largly on what visualization you currenly have selected.
+
+Continuous color modes use a value's percentage relative to min and max to interpolate a color.
```

Comment:
```suggestion
Continuous color modes use the percentage a value relative to min and max to interpolate a color.
```

Reply:
This reads incorrectly to me, missing an "of" 

> Continuous color modes use the percentage of a value relative to min and max to interpolate a color.